### PR TITLE
Jr/basic emergency screen

### PR DESCRIPTION
**Pull request by Coolpenguinz**
This is the basic emergency suite with no rights card or calling functionality.

### CODE REVIEW

File: js/components/EmergencyButton.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text, StyleSheet } from 'react-native';
+import Proptypes from 'prop-types';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import styles from '../styles/textStyles';
+// import { useTranslation } from 'react-i18next';
```
Comment (kiwansim) [inline js/components/EmergencyButton.js:6]: remove these comments!

File: js/components/EmergencyButton.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+export default function EmergencyButton({ title, onPress }) {
+  return (
+    <TouchableOpacity
+      style={buttonStyles.EmergencyButton}
+      onPress={() => onPress()}
+    >
+      <Text style={[styles.h2, { color: 'white' }, { textAlign: 'center' }]}>
```
Comment (kiwansim) [inline js/components/EmergencyButton.js:14]: import color from style file

File: js/components/EmergencyButton.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+EmergencyButton.propTypes = {
+  title: Proptypes.string.isRequired,
+  onPress: Proptypes.func.isRequired,
+};
+
+const buttonStyles = StyleSheet.create({
+  EmergencyButton: {
+    backgroundColor: '#333333',
```
Comment (kiwansim) [inline js/components/EmergencyButton.js:28]: import color file here too

File: js/components/EmergencyButton.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (kiwansim) [inline js/components/EmergencyButton.js:1]: Use the primary button component in place of importing EmergencyButton

Comment (JacobJaffe) [inline js/components/EmergencyButton.js:1]: Can we use primary button?

File: js/navigation/MainTabs.js [hunk 5]
```diff
@@ -2,25 +2,31 @@
...
+
+const FakeScreen = () => null;
 
-const MainTabs = () => {
+const AppTabs = () => {
   const { t } = useTranslation();
   return (
     <Tabs.Navigator
-      name={routes.mainTabs}
+      // tabBar={(props) => <MyTabBar {...props} />}
```
Comment (JacobJaffe) [inline js/navigation/MainTabs.js:28]: remove comment

File: js/navigation/MainTabs.js [hunk 6]
```diff
@@ -79,12 +85,20 @@ const MainTabs = () => {
         }}
       />
       <Tabs.Screen
+        listeners={({ navigation }) => ({
+          tabPress: (e) => {
+            // Prevent default action
+            e.preventDefault();
+
+            navigation.navigate('emergency_modal');
```
Comment (JacobJaffe) [inline js/navigation/MainTabs.js:93]: can this be `routes.emergencyModal`?

File: js/navigation/MainTabs.js [hunk 7]
```diff
@@ -79,12 +85,20 @@ const MainTabs = () => {
         }}
       />
       <Tabs.Screen
+        listeners={({ navigation }) => ({
+          tabPress: (e) => {
+            // Prevent default action
```
Comment (kiwansim) [inline js/navigation/MainTabs.js:90]: Comment what the hack is doing and explain

File: js/navigation/MainTabs.js [hunk 8]
```diff
@@ -79,12 +85,20 @@ const MainTabs = () => {
         }}
       />
       <Tabs.Screen
+        listeners={({ navigation }) => ({
```
Comment (JacobJaffe) [inline js/navigation/MainTabs.js:88]: Can you explain this in comments?

File: js/navigation/MainTabs.js (hunk 5 above)
Comment (kiwansim) [inline js/navigation/MainTabs.js:28]: Take out all the comments

File: js/screens/EmergencyScreen.js [hunk 9]
```diff
@@ -1,26 +1,75 @@
...
+import EmergencyButton from '../components/EmergencyButton';
 
-export default function Alerts() {
+export default function EmergencyScreen({ navigation }) {
+  const { t } = useTranslation();
   return (
-    <View style={styles.container}>
-      <Text style={styles.text}>{'This is the Emergency pop-up!'}</Text>
+    <View style={localstyles.container}>
+      <View>
```
Comment (kiwansim) [inline js/screens/EmergencyScreen.js:16]: remove extra view tag

File: js/screens/EmergencyScreen.js [hunk 10]
```diff
@@ -1,26 +1,75 @@
...
+          <View style={localstyles.titleRow}>
+            <MaterialCommunityIcons
+              name="alert-outline"
+              color="orange"
+              size={30}
+            />
+            <Text style={styles.h1}>{t('emergency_toolkit')}</Text>
+          </View>
+          <View style={localstyles.buttonStack}>
+            <EmergencyButton
```
Comment (kiwansim) [inline js/screens/EmergencyScreen.js:33]: Pass in styling prop to Primary Button component and enable dark mode!

File: js/screens/EmergencyScreen.js [hunk 11]
```diff
@@ -1,26 +1,75 @@
...
 
-const styles = StyleSheet.create({
+EmergencyScreen.propTypes = {
+  navigation: PropTypes.shape({
+    navigate: PropTypes.func.isRequired,
+    goBack: PropTypes.func.isRequired,
+  }).isRequired,
+};
+
+const localstyles = StyleSheet.create({
```
Comment (kiwansim) [inline js/screens/EmergencyScreen.js:55]: camelCase this

**Pull request merged**